src/utl/src/Utl.tcl

```diff
@@ -0,0 +1,132 @@
+global MAN_PATH 
+set MAN_PATH ""
+
+sta::define_cmd_args "man" { name\
+                            [-manpath manpath]\
+                            [-no_query]}
+proc man { args } {
+  sta::parse_key_args "man" args \
+    keys {-manpath} flags {-no_query}
+  
+  set name [lindex $args 0]
+  set DEFAULT_MAN_PATH "/usr/local/share/man/cat"
```

Comment:
shouldn't this be the directory of the `openroad bin/../share/man/cat` as the default?